nemo/collections/nlp/modules/common/megatron/megatron_bert.py

```diff
@@ -66,13 +64,14 @@ def __init__(self, model_name, config, vocab_file):
         )
 
         # read Megatron arguments back
-        args = get_args()
+        get_args()
```

Comment:
What does it do?

Reply:
It initializes Megatron arguments with megatron module